packages/gatsby-source-nacelle/src/source-nodes/index.js

```diff
@@ -79,19 +60,17 @@ module.exports = async function ({
 
           newNodeCount += 1;
         } else {
-          touchNode({
-            nodeId: `Nacelle${dataTypeUpper}-${entry[uniqueIdProperty]}`
-          });
+          touchNode(`Nacelle${dataType}-${entry[uniqueIdProperty]}`);
```

Comment:
The `nodeId` way was throwing an error for me. Going back to using just the string worked. But since we are bumping this to Gatsby v4, `touchNode` requires the entire node to to be passed in, so I changed it to use `getNode` before passing it to `touchNode`.